### PR TITLE
PD-702 Segment identify calls include `default_address` rather than `address`

### PR DESCRIPTION
**Pull request by edwardupton**

### CODE REVIEW

File: src/segmentTracker/helpers/identifyCustomer.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import { CustomWindow } from '../../..';
+import { getCookie } from '../../common/getCookie';
+import { setCartOnlyAttributes } from '../../common/setClientID';
+import { addUserIdForCustomer } from './addUserIdForCustomer';
+
+declare let window: CustomWindow;
```
Comment (lyykfi) [inline src/segmentTracker/helpers/identifyCustomer.ts:6]: we do that in many places: `declare let window: CustomWindow;`
I think we could create a file main.d.ts and add the CustomWindow declaration there.

Only what we need todo after to add to package.json this string: "typings": "main.d.ts"

File: src/segmentTracker/helpers/identifyCustomer.ts [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+export const identifyCustomer = (): void => {
+	const { customer, cookiesToTrack = [] } = window.LittledataLayer;
+
+	if (!customer) return;
+
+	const cookieTraits: any = {};
```
Comment (lyykfi) [inline src/segmentTracker/helpers/identifyCustomer.ts:13]: any  -> Record<string, string>

File: src/segmentTracker/helpers/identifyCustomer.ts (hunk 1 above)
Comment (edwardupton) [reply src/segmentTracker/helpers/identifyCustomer.ts:6]: But isn't that what 'index.d.ts' does? Can you make the change on another PR as it's unrelated to this?

**Review by lyykfi — APPROVED** [review]

**Comment by lyykfi** [comment]
@edwardupton could you fix conflicts pls?

**Pull request merged**